content/gettingstarted/index.adoc

```diff
@@ -16,129 +16,132 @@ toc::[]
 
-Once this is done, Kiali will be deployed and running in Kubernetes with an Ingress set up.
+If the authentication strategy of "login" is selected, a secret containing Kiali login credentials is required to be deployed along with Kiali. In this case, the install script will prompt you to enter a username and passphrase for the credentials that you want users to enter in order to log in successfully to Kiali. The install script will store those credentials in a secret that is deployed in the same namespace where Kiali is installed.
 
-If you deployed in minikube, you can go directly to the Kiali UI at the URL `https://[minikube-ip]/kiali/console` where `[minikube-ip]` is what is reported when you run the command: `minikube ip`.
+icon:bullhorn[size=1x]{nbsp} If you told the install script to not create a Kiali CR (and thus not have Kiali installed yet) via the OPERATOR_INSTALL_KIALI=false environment variable, you are responsible for creating this secret if you wish to install Kiali with the authentication strategy of "login". A secret is not required if your authentication strategy is not "login". The following command is a simple way to create a secret for Kiali whose username is "admin" and passphrase is "admin":
+[source,bash]
+----
+oc create secret generic kiali -n istio-system --from-literal "username=admin" --from-literal "passphrase=admin"
+----
 
-== Uninstall
+Once the Kiali Operator has installed Kiali and the Kiali pod has successfully started, you can begin to access Kiali.
 
-To uninstall Kiali is simple - just delete the objects that have the label `app=kiali` from your cluster.
+If on OpenShift, you can easily find the URL to access Kiali by going to the OpenShift console and navigating to the `istio-system` project's Overview page:
 
-To uninstall from OpenShift:
+[#img-openshift]
+.OpenShift console
+image::os-console.png[OpenShift console]
+{nbsp} +
+
+In this case, the Kiali URL is `http://kiali-istio-system.192.168.64.13.nip.io`. In your case this will be a different IP.
+
+You can also use the `oc` command to determine the base URL:
 
 [source,bash]
 ----
-oc delete all,secrets,sa,configmaps,deployments,ingresses,clusterroles,clusterrolebindings,virtualservices,destinationrules,customresourcedefinitions,templates --selector=app=kiali -n istio-system
+oc get route -n istio-system -l app=kiali
 ----
 
-To uninstall from Kubernetes:
+If on Kubernetes, access Kiali over the Ingress URL.
+
+== Uninstall
+
+To uninstall Kiali is simple - just delete the Kiali CR. This will trigger the Kiali Operator to uninstall Kiali:
 
 [source,bash]
 ----
-kubectl delete all,secrets,sa,configmaps,deployments,ingresses,clusterroles,clusterrolebindings,virtualservices,destinationrules,customresourcedefinitions --selector=app=kiali -n istio-system
+oc delete kiali kiali -n kiali-operator
```

Comment:
Should kiali be listed twice?

Reply:
yes... the command is "oc delete <the type of resource> <the name of the instance to be deleted> ..." in this case the resource type and the resource name are the same -- "kiali"

Reply:
Ah, I was wondering if it was a typo or something like that, two resources with the same name.  Thanks!